code/src/components/TodoList.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import moment from 'moment'
+
+import todos from '../reducers/todos'
+
+const TaskItem = styled.div `
+  box-shadow: 0 3px 4px #d6d2d2, 0 -0.5px 1px #d6d2d2;
+  margin-bottom: 9px;
+  padding: 10px 10px;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const DeleteButton = styled.button`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  border: none;
+  background-color: #9c9c9c;
+  color: #fff; 
+  font-size: 22px;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 10px;
+  padding-bottom: 3px;
+  transition: background-color 0.4s;
+  &:hover {
+    background-color: #ea003a;
+    cursor: pointer;
+  }
+`
+
+const Timestamp = styled.p`
+  margin: 3px 0 0 40px;
+  font-size: 10px;
+  color: #767676;
+`
+
+const Checkmark = styled.input.attrs({type:'checkbox'}) `
+  appearance: none;
+  -webkit-appearance: none;
+  height: 25px;
+  width: 25px;
+  background-color: #d5d5d5;
+  border-radius: 5px;
+  cursor: pointer; 
+  display: flex;
+  align-items: center;
```

Comment:
Really good work with your checkboxes! I really like the way you styled them, and I know how tricky it can be to style those, so you should be proud!!